__mocks__/stories/accountHierarchy.ts

```diff
@@ -0,0 +1,155 @@
+const commerceSummary = {
+  totalOrderAmount: {
+    amount: 0,
+  },
+  orderCount: 0,
+  wishlistCount: 0,
+  visitsCount: 0,
+}
+
+const updatedAndCreatedBy = {
+  updateBy: 'tbd',
+  createBy: 'tbd',
+}
+
+const userFlags = {
+  isLocked: false,
+  isActive: true,
+  isRemoved: false,
+  acceptsMarketing: false,
+  hasExternalPassword: false,
+}
+
+const commonAccountDetails = {
+  isActive: true,
+  salesReps: [],
+  rootAccountId: 1004,
+  approvalStatus: 'Approved',
+  customerSet: 'default',
+  commerceSummary,
+  segments: [],
+  taxExempt: false,
+  taxId: '123456',
+  accountType: 'B2B',
+  migrationRequired: false,
+}
+
+export const accountHierarchy = {
```

Comment:
user the Type
rename file with [TypeName]Mock

Reply:
Done